py3status/docstrings.py

```
@@ -69,9 +70,15 @@ def core_module_docstrings(include_core=True, include_user=False, config=None):
         with open(path) as f:
             module = ast.parse(f.read())
             docstring = ast.get_docstring(module)
-            docstring = [
-                d for d in _from_docstring(str(docstring).strip().split('\n'))
-            ]
+
+            if format == 'md':
+                docstring = [
+                    d for d in _from_docstring_md(str(docstring).strip().split('\n'))
+                ]
+            elif format == 'rst':
+                docstring = [
+                    d for d in _from_docstring_rst(str(docstring).strip().split('\n'))
+                ]
```

maybe raise an exception on unsupported `else` ?